### ISSUE TITLE
New Pro Pack Alpha 360-degrees

### ISSUE THREAD

**Issue by amkuchta** [fluttercommunity/font_awesome_flutter#158]
The new v6 alpha contains an icon named `360-degrees` which causes some issues when generating new icon names ("`IconData` isn't a type.").  My suggestion is to add a key/value pair to the `nameAdjustments` map:

`"360-degrees": "threeSixtyDegrees"`

**Comment by RegisSaffi** [comment]
Hey, I am facing the same issue, how can I resolve it?

**Comment by michaelspiss** [comment]
Hi  @RegisSaffi, in `tool/generate_font.dart` and `tool/generate_example.dart` there is a Map
```dart
const Map<String, String> nameAdjustments = {
  "500px": "fiveHundredPx",
  "1": "one",
  "2": "two",
  "3": "three",
  "4": "four",
  "5": "five",
  "6": "six",
  "7": "seven",
  "8": "eight",
  "9": "nine",
  "0": "zero",
};
```
just add `"360-degrees": "threeHundredSixtyDegrees",` after ther "zero" line in both files and you should be fine

**Comment by RegisSaffi** [comment]
Hey, thanks.

So, how can I use DuoTone icons in my project? I can only use normal Icons now.

**Comment by michaelspiss** [comment]
Please have a look at the [readme's pro icons section](https://github.com/fluttercommunity/font_awesome_flutter#how-can-i-use-pro-icons). If you have at least one duotone icon in your icons.json, `FaDuotoneIcon` is enabled. For further information please have a look at the [duotone icon section](https://github.com/fluttercommunity/font_awesome_flutter#duotone-icons). If you have any problems please open a new issue

**Comment by RegisSaffi** [comment]
Thanks @michaelspiss , I managed to do it, I had to fix dartfmt path issues in update.sh script which was causing problems.

**Comment by michaelspiss** [comment]
Fixed with version 9.1.0
